frontend/app/views/fragments/form/subscriberNumber.scala.html

```diff
@@ -3,12 +3,23 @@
 <fieldset class="fieldset">
 
     <div class="fieldset__heading">
-        <h2 class="fieldset__headline">Subscribers pay nothing for 6 months</h2>
+        <h2 class="fieldset__headline">Subscribers pay nothing<br /> for six months</h2>
```

Comment:
This was partly for aesthetic reasons and partly to fit Guardian style (six not 6).